src/Raven.Server/Documents/DocumentsStorage.Debug.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Raven.Server.ServerWide.Context;
+using Voron.Data.Tables;
+
+namespace Raven.Server.Documents
+{
+    public partial class DocumentsStorage
+    {
+        // Used to delete corrupted document from the JS admin console
+        public void DeleteDocumentByEtag(long etag)
```

Comment:
How do we prevent calling this ourselves from regular code? Should be under `.Debug` or `.EmergencyOperations` or something like that